tests/Hedgehog.Linq.Tests/NameTests.cs

```diff
@@ -36,8 +37,10 @@ public static IEnumerable<object[]> AllPublicMembers()
                     }
 
                     var mi = member as MethodInfo;
-                    if (mi != null &&
-                        (mi.IsSpecialName || mi.IsConstructor || mi.Name.StartsWith("get_")))
+                    if (mi != null && (
+                           mi.IsSpecialName
+                        || mi.IsConstructor
+                        || mi.Name.StartsWith("get_")))
```

Comment:
It would be better to just extract a method for the expression on the right of `&&`.